crates/nu-cmd-dataframe/src/dataframe/utils.rs

```diff
@@ -1,15 +1,16 @@
 use nu_protocol::{FromValue, ShellError, Value};
 
 pub fn extract_strings(value: Value) -> Result<Vec<String>, ShellError> {
+    let span = value.span();
     match (
-        <String as FromValue>::from_value(&value),
-        <Vec<String> as FromValue>::from_value(&value),
+        <String as FromValue>::from_value(value.clone()),
+        <Vec<String> as FromValue>::from_value(value),
```

Comment:
Mhh I think this evaluation of both paths here is a bit unfortunate, leading us to clone a potentially clone a large list just to fail the string test.

But that is no regression of your change.